aws-xray-recorder-sdk-aws-sdk-instrumentor/build.gradle.kts

```diff
@@ -4,7 +4,7 @@ plugins {
 }
 
 dependencies {
-    compileOnly(project(":aws-xray-recorder-sdk-aws-sdk"))
+    implementation(project(":aws-xray-recorder-sdk-aws-sdk"))
```

Comment:
We actually [don't have a dependency](https://github.com/aws/aws-xray-sdk-java/blob/master/aws-xray-recorder-sdk-aws-sdk-instrumentor/pom.xml) on the `aws-xray-recorder-sdk-aws-sdk` module currently. But that should be fine since the customers who are depending on both right now would just have it deduped. We should patch the [docs](https://docs.aws.amazon.com/xray/latest/devguide/xray-sdk-java-awssdkclients.html) when this is released.